README.md

```diff
@@ -1,5 +1,5 @@
 # HubL Visual Studio Code Language Extension
```

Comment:
Let's officially rename it in the readme HubSpot Visual Studio Code Extension or HubSpot VS Code Extension.
While it primarily is used to make HubL easier to work with, it provides other utilities like JSON snippets for fields.
I could see the feature list growing to encompass more HubSpotty things.